charm-slurmctld/src/interface_slurmd.py

```diff
@@ -152,6 +154,24 @@ def get_slurmd_info(self) -> list:
 
         return ensure_unique_partitions(partitions)
 
+    def set_nhc_params(self, params: str = ""):
+        """Send NHC parameters to all slurmd."""
```

Comment:
Can you help me understand why `nhc-params` is not a charm config of the `slurmd` charm?